--- ./alsa-monitor.conf	2021-10-07 22:31:04.419697180 +0300
+++ ./alsa-monitor.conf.new	2021-10-07 22:31:30.276406530 +0300
@@ -90,7 +90,7 @@
             }
             {
                 # Matches all sinks.
-                node.name = "~alsa_output.*"
+                node.name = "~alsa_output.usb*"
             }
         ]
         actions = {
@@ -113,11 +113,13 @@
                 #latency.internal.rate = 0            # internal latency in samples
                 #latency.internal.ns   = 0            # internal latency in nanoseconds
 
-                #api.alsa.period-size   = 1024
-                #api.alsa.headroom      = 0
+                # here
+                api.alsa.period-size   = 1024
+                
+                api.alsa.headroom      = 0
                 #api.alsa.start-delay   = 0
                 #api.alsa.disable-mmap  = false
-                #api.alsa.disable-batch = false
+                api.alsa.disable-batch = true
                 #api.alsa.use-chmap     = false
                 #iec958.codecs          = [ PCM DTS AC3 MPEG MPEG2-AAC EAC3 TrueHD DTS-HD ]
             }
