--- pipewire-pulse.conf	2021-10-07 23:53:37.381517474 +0300
+++ pipewire-pulse.conf.new	2021-10-07 23:54:14.731614529 +0300
@@ -41,12 +41,18 @@
                 #"tcp:[::]:9999"                    # IPv6 on all addresses
                 #"tcp:127.0.0.1:8888"               # IPv4 on a single address
             ]
-            #pulse.min.req          = 256/48000     # 5ms
+            
+            # here
+            pulse.min.req          = 256/48000     # 5ms
+            
             #pulse.default.req      = 960/48000     # 20 milliseconds
             #pulse.min.frag         = 256/48000     # 5ms
             #pulse.default.frag     = 96000/48000   # 2 seconds
             #pulse.default.tlength  = 96000/48000   # 2 seconds
-            #pulse.min.quantum      = 256/48000     # 5ms
+            
+            # here
+            pulse.min.quantum      = 256/48000     # 5ms
+            
             #pulse.default.format   = F32
             #pulse.default.position = [ FL FR ]
             # These overrides are only applied when running in a vm.
